content/blog/2020-01-ggplot2-3-3-0/index.Rmarkdown

```diff
@@ -0,0 +1,231 @@
+---
+title: "ggplot2 3.3.0"
+author: "Thomas Lin Pedersen"
+date: '2020-01-07'
+description: |
+  The next iteration of ggplot2 has just been released on CRAN, packed with new 
+  features and bug fixes. Read all about what's new here.
+photo:
+  author: Steve Harvey
+  url: https://unsplash.com/photos/xWiXi6wRLGo
+slug: ggplot2-3-3-0
+categories: ["package"]
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE, 
+  comment = "#>"
+)
+```
+
+We're so happy to announce the release of ggplot2 3.3.0 on CRAN. ggplot2 is a system for declaratively creating graphics, based on *The Grammar of Graphics*. You provide the data, tell ggplot2 how to map variables to aesthetics, what graphical primitives to use, and it takes care of the details.
+
+This release is a substantial release with both big new features and internal rewrites. It contains most of the work Dewey Dunnington did as part of his internship at RStudio, along with contributions from many others. Dewey will continue working on ggplot2 as part of the core team, and we are very lucky to have him.
+
+## New features
+The last ggplot2 release was mainly a performance-oriented release. This one, on the other hand is packed with features, big and small. There is also a slew of bug-fixes, but as that make for less interesting reading, please consult the NEWS.md file for those. Further, we have now removed reshape2 from the dependencies, which further removes the indirect dependencies on plyr, stringi and stringr. We have been shaving off dependencies in ggplot2 over a range of releases, and with the removal of reshape2 we are now close to being as lean as we believe is possible.
```

Comment:
```suggestion
The last ggplot2 release was primarily performance-oriented. This one, on the other hand is packed with features, big and small. There are also a slew of bug-fixes which you can read about in the [NEWS.md file](https://github.com/tidyverse/ggplot2/blob/master/NEWS.md). Further, we have now removed reshape2 from the dependencies, which removes the indirect dependencies on plyr, stringi, and stringr. We have been shaving off dependencies in ggplot2 over a range of releases, and with the removal of reshape2, we are now close to being as lean as we believe is possible.
```

Reply:
The diff above isn't showing up well. I tightened up the sentences a little bit, feel free to stet me.